test/e2e/suite/cases/stress.go

```diff
@@ -108,10 +108,6 @@ var _ = framework.CasesDescribe("Normal CSI behaviour", func() {
 
 		err = f.Helper().CertificateKeyMatch(cr, certData, keyData)
 		Expect(err).NotTo(HaveOccurred())
-
-		By("Ensure the certificate key pair and metadata file exists in the local data directory and matches that in the CertificateRequest")
```

Comment:
I removed all references to these checks as I'm not confident we need them - wdyt @JoshVanL?

Reply:
Yes, I think they are a bit redundant. We are already checking the paths from the driver’s data-dir, no need to check it on the actual node too.